pkgs/top-level/all-packages.nix

```
@@ -24497,7 +24497,9 @@ in
 
   thepeg = callPackage ../development/libraries/physics/thepeg { };
 
-  yoda = callPackage ../development/libraries/physics/yoda { };
+  yoda = callPackage ../development/libraries/physics/yoda {
+    pythonPackages = python2Packages;
```

python2 is EOL, please use python3Packages if possible

---

using my other suggestions:
```suggestion
    python = python3;
```

---

I will be switching `yoda` to use `python3` by default, but that, probably, can not be done until I get `rivet` version 3 working.